0xPolygonMiden/miden-client
# refactor: normalize transaction script data

closes #97

One doubt I have is if the transaction script hash depends on its inputs or not.
- This was checked and it does not depend on its inputs. It's just a hash of the script's code block

## Steps to test

1. Clone the miden node repo and generate the genesis data (it might be wise to delete the store db files as well).
2. ~if #95 is still not merged and its commints are not in this pr, you'll have to checkout to `mFragaBA-load-genesis-and-accounts` to run `rm target/release/store.sqlite3 && cargo run --release --features testing,concurrent load-accounts --accounts-path <path to your miden node repo>/accounts`. After that you can checkout to this branch again~
3. sync the node with `cargo run --release --features testing,concurrent sync-state -s`
4. create a new mint transaction with `cargo run --release --features testing,concurrent transaction new mint 0x4f0bf379a1328b70 0x92ec89c48c77256b 1000`
5. list and check that the new transaction appears with: `cargo run --release --features testing,concurrent transaction list`

you can also check with `sqlite3` that the transaction_scripts table has rows and the transactions table has rows as well
